src/main/java/lotto/enums/LottoWinnerPrice.java

```diff
@@ -0,0 +1,43 @@
+package lotto.enums;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum LottoWinnerPrice {
+    none_rank(0, 0),
+    fourth_rank(3, 5000),
+    third_rank(4, 50000),
+    second_rank(5, 1500000),
+    first_rank(6, 2000000000);
+
+    private final int matchedCount;
+    private final int price;
+
+    LottoWinnerPrice(int matchedCount, int price) {
+        this.matchedCount = matchedCount;
+        this.price = price;
+    }
+
+    public int getMatchedCount() {
+        return matchedCount;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public static int getPrice(int matchedCount) {
+        return Arrays.stream(LottoWinnerPrice.values())
+                .filter(s -> s.matchedCount == matchedCount)
```

Comment:
```suggestion
                .filter(price -> price.matchedCount == matchedCount)
```
s 가 무엇인지 모를 것 같아요.
stream 사용시 내부 변수에도 의미를 부여하면 좋을 것 같습니다!

Reply:
변수이름 구체화했습니다!